trestle/core/control_writer.py

```diff
@@ -76,42 +68,21 @@ def _add_control_statement(
         if print_group_title:
             group_name = ' \[' + group_title + '\]'
 
-        if capitalize_title:
-            control_id = control_id.upper()
-            group_name = group_name.title()
-            control_title = control_title.upper()
-
         title = f'{control_id} -{group_name} {control_title}'
 
         header_title = 'Control Statement'
-        if sections_dict and sections_dict['statement']:
-            header_title = sections_dict['statement']
         self._md_file.new_header(level=1, title=title)
         self._md_file.new_header(level=2, title=header_title)
         self._md_file.set_indent_level(-1)
         self._add_part_and_its_items(control, 'statement', 'item')
         self._md_file.set_indent_level(-1)
```

Comment:
In my current PR I made statement a const, so it will need to be resolved in merge.